docs/source/logging/index.md

```diff
@@ -1,19 +1,182 @@
+```
+
+## How to customise Kedro logging
+
+To customise logging in your Kedro project, you need to specify the path to a project-specific logging configuration file. Change the environment variable `KEDRO_LOGGING_CONFIG` to override the default logging configuration. Point the variable instead to your project-specific configuration, which we recommend you store inside the project's`conf` folder, and name `logging.yml`.
+
+For example, you can set `KEDRO_LOGGING_CONFIG` by typing the following into your terminal:
+
+```bash
+export KEDRO_LOGGING_CONFIG=<project_root>/conf/logging.yml
+```
+
+After setting the environment variable, any subsequent Kedro commands use the logging configuration file at the specified path.
 
 ```{note}
-Providing project-side logging configuration is entirely optional. You can delete the `conf/base/logging.yml` file and Kedro will run using the framework's built in configuration.
+If the `KEDRO_LOGGING_CONFIG` environment variable is not set, Kedro will use the [default logging configuration](https://github.com/kedro-org/kedro/blob/main/kedro/framework/project/default_logging.yml).
 ```
 
-Framework-side and project-side logging configuration are loaded through subsequent calls to [`logging.config.dictConfig`](https://docs.python.org/3/library/logging.config.html#logging.config.dictConfig). This means that, when it is provided, the project-side logging configuration typically _fully overwrites_ the framework-side logging configuration. [Incremental configuration](https://docs.python.org/3/library/logging.config.html#incremental-configuration) is also possible if the `incremental` key is explicitly set to `True` in your project-side logging configuration.
+### How to show DEBUG level messages
+To see `DEBUG` level messages, change the level of logging in your project-specific logging configuration file (`logging.yml`). We provide a `logging.yml` template:
+
+<details>
+<summary><b>Click to expand the <code>logging.yml</code> template</b></summary>
+<code>
 
+```yaml
+version: 1
 
-```{toctree}
-:hidden:
+disable_existing_loggers: False
 
-logging
+formatters:
+  simple:
+    format: "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
+
+handlers:
+  console:
+    class: logging.StreamHandler
+    level: INFO
+    formatter: simple
+    stream: ext://sys.stdout
+
+  info_file_handler:
+    class: logging.handlers.RotatingFileHandler
+    level: INFO
+    formatter: simple
+    filename: info.log
+    maxBytes: 10485760 # 10MB
+    backupCount: 20
+    encoding: utf8
+    delay: True
+
+  rich:
+    class: kedro.logging.RichHandler
+    rich_tracebacks: True
+    # Advance options for customisation.
+    # See https://docs.kedro.org/en/stable/logging/logging.html#project-side-logging-configuration
+    # tracebacks_show_locals: False
+
+loggers:
+  kedro:
+    level: INFO
+
+  your_python_package:
+    level: INFO
+
+root:
+  handlers: [rich]
+```
+</code>
```

Comment:
Not sure the code block should end here, the docs end up rendering as follows: 
![image](https://github.com/kedro-org/kedro/assets/90615669/30bfec0c-bbbd-47bc-ae54-02df1fa16b1e)

Reply:
![image](https://github.com/kedro-org/kedro/assets/18221871/b940869c-9de1-4e76-a79e-c3c1233e54a2)
 Before you expand the template it looks pretty normal. But I agree it looks weird after it get expanded. I will add a short line there.